apps/sms/js/drafts.js

```diff
@@ -265,8 +173,7 @@
         });
         draftIndex = new Map(records || []);
 
-        // Return list of threadless drafts
-        deferredDraftRequest.resolve(Drafts.byThreadId(null));
+        deferredDraftRequest.resolve();
```

Comment:
So here I see that nobody uses data passed here, so it looked like a waste, _but_ ```getAll``` is generator now and we can pass this ```lazy``` collection here without much penalty and use the result in ```InboxView.renderDrafts``` and this case we won't need to expose ```getAll``` publicly.